svc-bie-kafka/src/main/resources/application.yaml

```diff
@@ -0,0 +1,37 @@
+spring:
+  # Specify where Spring connects to RabbitMQ
+  rabbitmq:
+    host: "${RABBITMQ_PLACEHOLDERS_HOST:localhost}"
+    port: 5672
+    username: "${RABBITMQ_PLACEHOLDERS_USERNAME:guest}"
+    password: "${RABBITMQ_PLACEHOLDERS_USERPASSWORD:guest}"
+
+  # Specify where Spring connects to Kafka
+  kafka:
+    consumer:
+      group-id: "${BIE_KAFKA_PLACEHOLDERS_GROUP_ID:vro-bie-group}"
+    bootstrap-servers: "${BIE_KAFKA_PLACEHOLDERS_BROKERS:localhost:9092}"
+
+## Specify bie properties
+bie:
+  topic-map:
```

Comment:
Info about kafka topics can be found [here](https://github.com/department-of-veterans-affairs/VES/blob/master/research/Event%20Bus/Use%20Cases/Contention%20Events%20Use%20Case%20One%20Pager.md)